extensions/vertx-web-common/deployment/src/main/java/io/quarkus/vertx/web/common/deployment/InternalVertxWebRouterBuildItem.java

```diff
@@ -0,0 +1,18 @@
+package io.quarkus.vertx.web.common.deployment;
+
+import io.quarkus.builder.item.SimpleBuildItem;
+import io.quarkus.runtime.RuntimeValue;
+import io.vertx.ext.web.Router;
+
+public final class InternalVertxWebRouterBuildItem extends SimpleBuildItem {
```

Comment:
Why is it named `Internal`, that's the Router build item. We expect to have only one.

Reply:
Done. I didn't really want to expose it but why not.